README.md

```diff
@@ -1,2 +1,91 @@
 # queries
+
 Hosts reusable log queries which are built into a single queries.json file.
+
+## Query database structure
+
+```yaml
+queries:
+  pip-no-matching-distribution:  # also known as query "id", (sort key)
+    pattern: "ERROR: No matching distribution found for"  # required, can be list!
+    # optionals:
+    name: "PIP failed to find package"  # optional (aka 'reason'), id is used instead
+    files: []  # list of glob patterns, narrows down searching
+    regex: false  # optional, default is false
+    url: ... # str or list[str], issue links to lp, bz,...
+    category: ... # see Categories
+    tags: []  # see Tags
```

Comment:
so "tags" or "elastic-tags"?

Reply:
At this moment I do not see a reason to add extra verbosity but if that proves to be a bad decision in the future, I will do the rename myself.

The spec format is not very important because we have only one file in a single repository to deal with, so we can change the **source** spec at any time, as consumers will load the compiled form and not be affected.